project_release/project_release.cron.inc

```diff
@@ -230,21 +236,18 @@ function project_release_history_generate_project_xml(Node $project_node, $versi
  * @throws Exception
  * @return bool
  */
-function project_release_history_write_xml($xml, Node $project_node = NULL, $version_api = NULL) {
-  // Dublin core namespace according to http://dublincore.org/documents/dcmi-namespace/
-  $dc_namespace = 'xmlns:dc="http://purl.org/dc/elements/1.1/"';
+function project_release_history_write_xml(array $xml, Node $project_node = NULL, $version_api = NULL) {
   $full_xml = '<?xml version="1.0" encoding="utf-8"?>' . "\n";
+  $full_xml .= format_xml_elements($xml);
```

Comment:
:open_mouth: Nice one! I did not know you could do that! And it results in exactly the same XML?

Reply:
Per https://github.com/backdrop-contrib/project/issues/25#issuecomment-482881140 and follow-up, yes, yes it does.